src/helperFunctions/formHelper.ts

```diff
@@ -705,23 +705,10 @@ function checkAnswer(formdata: FormData) {
           break;
         case "date":
           const date = stepData.value as DateStep;
-
-          const day = dayNumber[date.day as number] || date.day
-
-          let dateDay = ""
-          if(day === 1 || day === 21 || day === 31) {
-            dateDay = `${day}st`
-          } else if(day === 2 || day === 22) {
-            dateDay = `${day}nd`
-          } else if(day === 3 || day === 23) {
-            dateDay = `${day}rd`
-          } else {
-            dateDay = `${day}th`
-          }
-
-          const dateMonth: string = date?.month?.toString() || ""
+           const newDate = formatDate(`${date.month}/${date.day}/${date.year}`)
```

Comment:
I've not checked out this PR to test, but just checking whether this produces the right dates? 
Because UK dates are usually in `day-month-year` format, not `month-day-year`.
Not sure if this would confuse the formatter or if it's clever enough to figure it out?

Reply:
Hi Kiran, if I send day/month/year to formDate it returns month day and year or invalid date with day is bigger than 12

Reply:
Thanks for checking Gabi,
Maybe the safest thing would be to create the newDate in ISO format, so `YYYY-MM-DD` should always work, I think

Reply:
@gabimartini  I don't think we need to do all this.

in the stringHelper, change day: from `numeric` to `day: "2-digit",`

this will format all `day` dates using formatDate to have 01-09 even if we type in 1-9

Reply:
I think it's fixed :)